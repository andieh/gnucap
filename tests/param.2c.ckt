spice

*bm_cond parser is broken.

E1 1 0 0 0 dc a ac b
v1 1 0 dc a ac b
.list
.end
