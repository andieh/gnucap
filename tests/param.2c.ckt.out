*bm_cond parser is broken.
